### ISSUE TITLE
Error testing web routes without using WithoutMiddleware trait

### ISSUE THREAD

**Issue by llstarscreamll** [Porto-SAP/Hello-API#74]
Testing web routes without the mentioned trait causes this error:

```bash
1) App\Containers\Authentication\UI\WEB\Tests\Functional\UserLoginTest::testUserLogin
InvalidArgumentException: Expecting a DOMNodeList or DOMNode instance, an array, a string, or null, but got "Illuminate\View\View".

/home/user/Code/hello/vendor/symfony/dom-crawler/Crawler.php:123
/home/user/Code/hello/vendor/symfony/dom-crawler/Crawler.php:70
/home/user/Code/hello/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/InteractsWithPages.php:96
/home/user/Code/hello/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/InteractsWithPages.php:61
/home/user/Code/hello/app/Containers/Authentication/UI/WEB/Tests/Functional/UserLoginTest.php:22
```

If the test in question uses the trait, everything goes green, but these behavior isn't convenient. Disabling middlewares sometimes on some tests is useful, but it isn't if they are always disabled.

**Comment by Mahmoudz** [comment]
@llstarscreamll Yep sometimes turning off middleware for some tests is useful, if you found a nice solution for this please do share it. Thanks

**Comment by Mahmoudz** [comment]
closed by #116
